lapis/src/main/kotlin/org/genspectrum/lapis/openApi/OpenApiDocs.kt

```diff
@@ -347,16 +347,29 @@ private fun filterFieldSchema(fieldType: SequenceFilterFieldType) =
             Schema<String>().anyOf(
                 listOf(
                     nullableStringSchema(fieldType.openApiType),
-                    nullableStringArraySchema(fieldType.openApiType),
+                    logicalOrArraySchema(nullableStringSchema(fieldType.openApiType)),
+                ),
+            )
+
+        is SequenceFilterFieldType.StringSearch ->
+            Schema<String>().anyOf(
+                listOf(
+                    nullableStringRegexSchema(fieldType.associatedField),
+                    logicalOrArraySchema(nullableStringRegexSchema(fieldType.associatedField)),
                 ),
             )
 
         else -> nullableStringSchema(fieldType.openApiType)
     }
 
-private fun nullableStringSchema(type: String) = Schema<String>().type(type).nullable(true)
+private fun nullableStringRegexSchema(associatedField: SequenceFilterFieldName) =
+    nullableStringSchema("string")
+        .description(
+            "A regex pattern close to PCRE for filtering '$associatedField'. " +
```

Comment:
In the docs I went with the same formulation as the google docs
```suggestion
            "A regex pattern (subset of PCRE) for filtering '$associatedField'. " +
```